app/data/DataStores.scala

```diff
@@ -2,29 +2,20 @@ package data
 
 import com.gu.atom.data._
 import com.gu.atom.publish._
-import com.gu.contentatom.thrift.{Atom, AtomData}
-import com.gu.contentatom.thrift.atom.media.MediaAtom
+import com.gu.contentatom.thrift.Atom
 import com.gu.media.pluto.PlutoProjectDataStore
 import com.gu.media.upload.UploadsDataStore
 import com.gu.media.{CapiPreviewAccess, PlutoDataStore}
-import com.gu.scanamo.DynamoFormat
 import util.atom.MediaAtomImplicits
-import com.gu.scanamo.scrooge.ScroogeDynamoFormat._
 import model.commands.CommandExceptions._
 import util.AWSConfig
 
-import scala.reflect.ClassTag
-
 class DataStores(aws: AWSConfig, capi: CapiPreviewAccess) extends MediaAtomImplicits {
   import cats.syntax.either._ // appears unused but is required to make the data stores compile
```

Comment:
Actually, as an alternative, I can also suggest importing cats everywhere 😆 That also fixed lots of issues...